### PR TITLE
add basic / example Dockerfile

### PR DESCRIPTION
**Pull request by grosser**
@atmos I'm thinking this should be good enough for most .... and for the rest a nice starting point

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM ubuntu
+
+RUN apt-get update && apt-get install -yq nodejs npm
```
Comment (atmos) [inline Dockerfile]: Can we specify the correct version of node that's in the package.json file?

Comment (grosser) [inline Dockerfile]: this will install something that is newer ... is that not good enough ?
(I'd hope it won't be downgraded in newer versions of ubuntu)

**Comment by atmos** [comment]
Can you throw some docs in the readme explaining intended startup/destroy behavior for it?

**Comment by grosser** [comment]
added build and run command, is that what you had in mind ?

**Comment by grosser** [comment]
looking good / need anything else ?

**Comment by atmos** [comment]
I keep not getting around to it. Sorry, I'll try to take a look this week.

**Comment by grosser** [comment]
@atmos  'this week' was in October, how about now ? :)

**Comment by atmos** [comment]
Basically every time I go to mess with this my docker setup locally is completely busted and I just close my laptop and go for a walk with my dogs.

**Comment by grosser** [comment]
Keep trying, your dogs will thank you ;)

maybe use docker on someone else's  laptop / server ...

**Comment by atmos** [comment]
I spent an hour or so reading the latest docs and got it working. I think I have some networking issues w/ the vmware fusion driver but it looks good. Thanks for the PR and thanks for bugging me to merge it. :bow:

**Comment by grosser** [comment]
Yeah :D

FYI running on virtualbox after I got stuck with vmware ...

**Pull request merged**